ntoskrnl/ps/process.c

```diff
@@ -1614,4 +1677,234 @@ NtOpenProcess(OUT PHANDLE ProcessHandle,
     /* Return status */
     return Status;
 }
+
+PEPROCESS NtpGetAdjacentProcess(const BOOLEAN is_backwards, const PEPROCESS current)
```

Comment:
`Ntp` does not make sense as a prefix -- anything that's a helper for an `NtXxx` function belongs to a specific kernel subsystem -- in this case, Ps, so it should be Psp.